frontend/src/layout/navigation/MainNavigation.tsx

```diff
@@ -271,6 +271,15 @@ export function MainNavigation(): JSX.Element {
                         hotkey="i"
                         tooltip="Answers to all your analytics questions."
                     />
+                    {featureFlags[FEATURE_FLAGS.SAVED_INSIGHTS] && (
+                        <MenuItem
+                            title="Saved Insights"
```

Comment:
It should just be "Insights" proper. You have to save to append to the list, so the "saved" portion is redundant.

Reply:
I can update the naming for sure but I wanted to build the "create a new insight" flow before removing that, otherwise it would be annoying to test with, and I wanted to minimize the amount of feature flags I put 😅

Reply:
Then maybe just not have the menu item in this PR?

Reply:
done